scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -40,6 +40,7 @@ import com.here.ort.model.ScannerDetails
 import com.here.ort.model.ScannerRun
 import com.here.ort.model.config.ScannerConfiguration
 import com.here.ort.model.mapper
+import com.here.ort.spdx.LICENSE_FILENAMES
```

Comment:
Somehow this felt like a good place for these constants, because it already has the `NON_LICENSE_FILENAMES` list which is somehow related. But I'm find with moving this back to utils (but not inside the `FileMatcher` then), as at least currently nothing in the spdx-utils module depends on it.

Reply:
I'll leave it up to you. I thought I'd share this because when I moved it out of SPDX one of the reasons was that I could not see how it relates to SPDX.